buildSrc/src/test/java/io/micronaut/guides/core/FilesTransferUtilityTest.java

```diff
@@ -0,0 +1,80 @@
+package io.micronaut.guides.core;
+
+import io.micronaut.json.JsonMapper;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@MicronautTest(startApplication = false)
+public class FilesTransferUtilityTest {
+
+  @Inject
+  FilesTransferUtility filesTransferUtility;
+
+  @Inject
+  JsonMapper jsonMapper;
+
+  @Inject
+  JsonSchemaProvider jsonSchemaProvider;
+
+
+  @Inject
+  GuideProjectGenerator guideProjectGenerator;
+
+  @Test
+  void testTransfer() throws Exception {
+    String outputPath = "build/tmp/test";
+    File outputDirectory = new File(outputPath);
+    outputDirectory.mkdir();
+    List<Guide> metadatas = new ArrayList<>();
+
+    String pathBase = "src/test/resources/guides/hello-base";
+    File fileBase = new File(pathBase);
+    GuideUtils.parseGuideMetadata(fileBase,"metadata.json", jsonSchemaProvider.getSchema(), jsonMapper).ifPresent(metadatas::add);
+    String path = "src/test/resources/guides/creating-your-first-micronaut-app";
+    File file = new File(path);
+    GuideUtils.parseGuideMetadata(file,"metadata.json", jsonSchemaProvider.getSchema(), jsonMapper).ifPresent(metadatas::add);
+
+    guideProjectGenerator.generate(outputDirectory,metadatas.get(0));
+    guideProjectGenerator.generate(outputDirectory,metadatas.get(1));
+
+    filesTransferUtility.transferFiles(outputDirectory,fileBase,metadatas.get(0));
+    filesTransferUtility.transferFiles(outputDirectory,file,metadatas.get(1));
+
+    assertTrue(new File(outputPath+"/creating-your-first-micronaut-app-gradle-groovy").exists());
```

Comment:
add spaces please 🙏. The whole framework uses spaces. 

Don't do: 

`outputPath+"/creating-your-first-micronaut-app-gradle-groovy"`

do: 

`outputPath + "/creating-your-first-micronaut-app-gradle-groovy"`